api.py

```diff
@@ -13,8 +13,10 @@ def __init__(self, client_id, client_secret):
         self.client_secret = client_secret
         self.token = None
 
+    """ Authenticates Client """
+    # Returns access token
     def _auth_client(self):
```

Comment:
(I know this isn't in the tutorial but...) It would be good to add return and parameter types to function definitions.